### PR TITLE
Adding code for IsUnique string

### PR DESCRIPTION
**Pull request by anilhariparamhans**

### CODE REVIEW

File: CrackingCodingInterview/vAnil/01 Array/03 URLify/URLify.cpp [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+					len += 2;
+			}
+		}
+	}
+}
+
+int main(int argc, char *argv[])
+{
+	char str[100] = "Hello World I am Here";
```
Comment (dimplenaresh1) [inline CrackingCodingInterview/vAnil/01 Array/03 URLify/URLify.cpp]: True string length is given to you and enough space are also given in the string to accommodate extra characters.

File: CrackingCodingInterview/vAnil/01 Array/02 CheckPermutation/CheckPermutation.cpp [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+using namespace std;
+
+bool IsPermutationString(string str1, string str2)
+{
+	bool result = false;
+
+	using Type = unordered_map<char, uint32_t>;
+
+	Type uMap1, uMap2;
+
```
Comment (dimplenaresh1) [inline CrackingCodingInterview/vAnil/01 Array/02 CheckPermutation/CheckPermutation.cpp]: check for length equality.

File: CrackingCodingInterview/vAnil/01 Array/03 URLify/URLify.cpp (hunk 1 above)
Comment (anilhariparamhans) [reply CrackingCodingInterview/vAnil/01 Array/03 URLify/URLify.cpp]: Review comment incorporated.

File: CrackingCodingInterview/vAnil/01 Array/02 CheckPermutation/CheckPermutation.cpp (hunk 2 above)
Comment (anilhariparamhans) [reply CrackingCodingInterview/vAnil/01 Array/02 CheckPermutation/CheckPermutation.cpp]: Review comment incorporated.

**Review by dimplenaresh1 — APPROVED** [review]

**Pull request merged**